pkgs/by-name/me/meson/package.nix

```diff
@@ -116,7 +122,13 @@ python3.pkgs.buildPythonApplication rec {
     ''test cases/linuxlike/14 static dynamic linkage''
     # Nixpkgs cctools does not have bitcode support.
     ''test cases/osx/7 bitcode''
-  ])
+  ] ++ lib.optionals stdenv.isFreeBSD [
+    # pch doesn't work quite right on FreeBSD, I think
+    ''test cases/common/13 pch''
+    # tests hardcode lib instead of libdata, which FreeBSD uses
+    ''test cases/common/44 pkgconfig-gen''
+    ''test cases/common/230 external project''
```

Comment:
Since we're patching anyway for this, can't we just change them too?

Reply:
Interestingly enough, applying the patch patch suggestion you had above fixes this outright. Very good call!